backend/organization/utility/project.py

```diff
@@ -39,6 +40,7 @@ def create_new_project(data: Dict) -> Project:
         project.is_draft = data['is_draft']
     if 'website' in data:
         project.website = data['website']
+    project.language = source_language
```

Comment:
Hey so language is a foreign key in project table. So I think you need to first get the language from `source_language` string and then set `project.language = language`

Reply:
Good point!